RIAssigner/data/PandasData.py

```diff
@@ -11,14 +11,15 @@ class PandasData(Data):
 
     def read(self, filename: str):
         self._data = read_csv(filename)
-        self._set_rt_index()
+        self._set_rt_index_n_position()
         self._set_carbon_number_index()
 
     def _set_carbon_number_index(self):
         self._carbon_number_index = get_first_common_element(self._data.columns, self._carbon_number_column_names)
 
-    def _set_rt_index(self):
+    def _set_rt_index_n_position(self):
         self._rt_index = get_first_common_element(self._data.columns, self._rt_column_names)
+        self._rt_position = self._data.columns.tolist().index(self._rt_index)

```

Comment:
This could get a more descriptive name such as `_init_rt_col_name_and_position` or `_init_rt_column_info`